spacemacs/packages.el

```diff
@@ -1677,14 +1677,19 @@ If ARG is non nil then `ag' and `pt' and ignored."
           (call-interactively (spacemacs//helm-projectile-do-search-find-tool
                                tools))))
 
+      (setq helm-projectile-sources-list '(helm-source-projectile-projects
+                                           helm-source-projectile-files-list
+                                           helm-source-recentf
+                                           helm-source-pp-bookmarks
+                                           helm-source-file-cache))
```

Comment:
Or, you could think it this command is a beginner command that combine many things that Spacemacs offer. He can simply use this magic command to discover whatever Spacemacs offers i.e. buffers, files, project files, projects, recent files, bookmarks, snippets, tags... and later if he finds it inconvenient, he can simply use the specialized command with its own key binding. We can write in the doc binding for each source so the user can switch gradually.